largest_palindrome_product.js

```diff
@@ -5,18 +5,49 @@
  *  @return {Object} an object containing the two factors used to produce
  *                   the palindromeNumber and the palindromeNumber itself.
  */
-module.exports = function(digits){
+module.exports = function(digits) {
   var factor_0 = 0;
   var factor_1 = 0;
   var palindromeNumber = 0;
 
   // do your work here
+  // going to work on function to check for more than just 2 and 3 digits. So find middle number (k) of product i * j and check if k + i = k - i.
 
-
+  n = Math.pow(10, digits);
+  for (i = 10; i < n; i++) {
+    for (j = 10; j < n; j++) {
+      var ijProduct = i * j;
+      var digit1 = Math.floor((ijProduct / 1) % 10);
```

Comment:
Try and do this so that you could handle any number of digits in the `ijProduct`